vds_api_client/api_cli.py

```diff
@@ -115,13 +115,17 @@ def download_if_unfinished(api_instance, n_jobs=1):
               default=vds_pass if vds_pass else pass_show,
               show_default=pass_show)
 @click.option('--impersonate', '-i',
-              help='User to impersonate')
+              help='Username to impersonate')
+@click.option('--host',
+              type=click.Choice(['maps', 'staging', 'test']),
+              help='Host to use for requests https://{host}.vandersat.com')
```

Comment:
I'm used to calling testing, staging, and production 'environments', so I would go for `--environment` or `--env`.

Reply:
Ok, that might be more intuitive indeed. However, I will only change it in the cli to stay backwards compatible